### ISSUE TITLE
MyTest.py 和 main.m  运行出问题

### ISSUE THREAD

**Issue by Songxinlei** [DengPingFan/PraNet#14]
运行MyTest,py的时候，出现以下警告：
`/home/sxl/anaconda3/lib/python3.6/site-packages/torch/nn/functional.py:3063: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  "See the documentation of nn.Upsample for details.".format(mode))
/home/sxl/anaconda3/lib/python3.6/site-packages/torch/nn/functional.py:3103: UserWarning: The default behavior for interpolate/upsample with float scale_factor changed in 1.6.0 to align with other frameworks/libraries, and now uses scale_factor directly, instead of relying on the computed output size. If you wish to restore the old behavior, please set recompute_scale_factor=True. See the documentation of nn.Upsample for details. 
  warnings.warn("The default behavior for interpolate/upsample with float scale_factor changed "
/home/sxl/anaconda3/lib/python3.6/site-packages/torch/nn/functional.py:2952: UserWarning: nn.functional.upsample is deprecated. Use nn.functional.interpolate instead.
  warnings.warn("nn.functional.upsample is deprecated. Use nn.functional.interpolate instead.")
/home/sxl/anaconda3/lib/python3.6/site-packages/scipy/misc/pilutil.py:106: RuntimeWarning: overflow encountered in multiply
  bytedata = (data - cmin) * scale + low`、
然后，过一会，也不出现什么，就结束了，之后，查看results里的图片，效果特别特别差，很多都是黑乎乎的一张图片，什么都没有。
2、运行main.m，会报以下错误
`(Dataset:CVC-ClinicDB; Model:PraNet) meanDic:NaN;meanIoU:NaN;wFm:NaN;Sm:NaN;meanEm:NaN;MAE:NaN;maxEm:NaN;maxDice:NaN;maxIoU:NaN;meanSen:NaN;maxSen:NaN;meanSpe:NaN;maxSpe:NaN.
Elapsed time is 0.011206 seconds.`
全是NAN
还望作者可以给解答下，是不是我哪里做错了？谢谢！@DengPingFan

**Comment by GewelsJI** [comment]
@Songxinlei  数值溢出，不知道您修改了哪一部分的代码，我在output后加了`sigmoid()`进行了数值的缩放，应该不会出现类似的问题，如果方便的话，请您提供一下相关的代码。感谢您的关注。

**Comment by Songxinlei** [comment]
en  我只是改了数据集的路径，其他的代码 我应该没有动。我通过邮件发给您那个 code吧

**Comment by GewelsJI** [comment]
Hi @Songxinlei!

This is my e-mail (gepengai.ji@gmail.com)

Best,
Ge-Peng.

**Comment by Songxinlei** [comment]
ok   thanks

**Comment by Pr0mlse** [comment]
你好，我运行MyTest.py可以正常对测试集进行预测，但是运行main.m时遇到了和你一样的问题，请问你解决了吗
